modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheProcessorActiveTxTest.java

```diff
@@ -30,12 +30,21 @@
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.IntStream.range;
 import static org.apache.ignite.testframework.GridTestUtils.assertThrows;
+import static org.apache.ignite.testframework.GridTestUtils.getFieldValue;
 
 /**
  * Tests that verify the execution of GridCacheProcessor operations
  * in active transactions.
  */
 public class GridCacheProcessorActiveTxTest extends GridCommonAbstractTest {
+    /**
+     * Format for displaying the cache name(s) and operation.
+     *
+     * @see GridCacheProcessor#CHECK_EMPTY_TRANSACTIONS_ERROR_MSG_FORMAT
```

Comment:
This link is highlighted by idea as this field is private

Reply:
![image](https://user-images.githubusercontent.com/2736390/88566788-32adf600-d03f-11ea-896e-7e4ef7553e32.png)